build.cake

```diff
@@ -227,7 +248,7 @@ Task("VSTestPrepare")
     .Does(() =>
 {
     var externalRoot = "./externals/";
-    var testAssembliesFolder = "./tests/VisualStudio.AndroidX.Migration/Test/Assemblies/";
+    var testAssembliesFolder = "./externals/test-assets/vs-tests/Assemblies/";
```

Comment:
Moved the external assemblies into the externals folder so everything is there and also so they don't get checked in accidentally.